### ISSUE TITLE
Flight table: Sorting issue when multiple flights in same day

### ISSUE THREAD

**Issue by snip** [skylines-project/skylines#411]
When a pilot publish multiple IGC files for a same day, in flight table when sorting by date (default sorting), flights are mixed for a given day.
It may be more suitable to sort flights by date and time by default.

Example: The 7th of September 2015 https://skylines.aero/flights/pilot/413

**Comment by pyrog** [comment]
The table is sorted by the selected column AND by score.

The sort seem to be done in the server side python script:
https://github.com/skylines-project/skylines/blob/master/skylines/frontend/views/flights.py#L100

**Comment by Turbo87** [comment]
@pyrog you are correct that this is currently the order in which things are displayed, but I think @snip has a point here that when viewing my own personal flight list I'd expect them in chronological order instead, so maybe we should adjust the sorting if the flight list of a pilot is requested.

**Comment by pyrog** [comment]
@Turbo87 I think this is more general. For humans, sorting by _date_ mean by _date_ **and** by _time_ 😉 

Also, the _time_ column is sortable but sorting by _date_ or _time_ have the same effect.
To avoid confusion, I suggest to put these two fields in the same column.

**Comment by Turbo87** [comment]
putting date and times in the same column has some side effects. for example if you look at the flights list for a specific date the "date" column is hidden, but the takeoff/landing times are shown. if you put both in the same column this will get more difficult.

maybe we should only sort by score if a specific date is requested and sort by date/time in all other cases?

**Comment by pyrog** [comment]
> putting date and times in the same column has some side effects. 

Ok. So, I suggest to remove the sortable attribute of the column _time_.

> maybe we should only sort by score if a specific date is requested and sort by date/time in all other cases?

Why not simply sort by _score_ only If the user select this column?

**Comment by Turbo87** [comment]
> Why not simply sort by score only If the user select this column?

because the flight list for specific dates should be sorted by score similar to how the OLC does it

> Ok. So, I suggest to remove the sortable attribute of the column time.

I'd personally prefer it if both are "sortable" even if they ultimately result in the same list.

**Comment by pyrog** [comment]
> because the flight list for specific dates should be sorted by score similar to how the OLC does it

Ok 

PS: I'm not a pilot: _OLC_ mean _On-Line Contest_ 😉

**Comment by Turbo87** [comment]
> PS: I'm not a pilot: OLC mean On-Line Contest 😉

yes, see http://www.onlinecontest.org/olc-2.0/gliding/daily.html

**Comment by lordfolken** [comment]
Closing as stale/wontfix in 2026 revitalization triage; reopen with fresh repro if still relevant.
